numpy/ma/tests/test_core.py

```diff
@@ -48,6 +49,12 @@
 pi = np.pi
 
 
+suppress_copy_mask_on_assignment = suppress_warnings()
+suppress_copy_mask_on_assignment.filter(
+    numpy.ma.core.MaskedArrayFutureWarning,
+    "setting an item on a masked array which has a shared mask will not copy")
+
```

Comment:
You know, I have no idea what that `MaskedArrayFutureWarning` means. In fact, I can't even find the message anywhere...